inst/run_orderly_demo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
I think the existing `create_orderly_demo` script is still in use by things other than OW (e.g. for testing in this repo), so I've left that as is and just added a new one for the usage we want in OW.

Reply:
Looks like `create_orderly_demo` script is used in test set in many places, like montagu-proxy, montagu-reporting-api, montagu-webapps. If we are not using `create_orderly_demo` in orderly-web any more then possibly worth backing out commit https://github.com/vimc/orderly/pull/320/commits/d7922bd892ee4cbe1be1e9169d1de89b823c05a1 from this PR to avoid potentially breaking the other components which use it

Reply:
Ok yep, let's do that.